website/content/docs/enterprise/license/faq.mdx

```diff
@@ -0,0 +1,91 @@
+---
+layout: docs
+page_title: Nomad Enterprise License FAQ
+description: >-
+  Frequently Asked Questions pertaining to Nomad Enterprise Licensing.
+---
+
+# Frequently Asked Questions (FAQ)
+
+This FAQ is for the license changes introduced in Nomad Enterprise version v1.1.0+ent. 
+Nomad Enterprise automatically load Nomad licenses when a Nomad server agent starts using Nomad Enterprise.
+
+## Q: Can I get a quick summary of the Nomad changes?
+
+Starting with Nomad Enterprise v1.1.0, the license enablement process is different.
+
+HashiCorp Enterprise servers will no longer start without a license. When the server instance initializes the license watcher, the server reads from either an environment variable or file. If the license is missing, invalid, or expired, the server will immediately exit. 
+This check is part of the server boot-up process.
+In previous versions of HashiCorp enterprise products, one server could distribute a license to other servers via the Raft protocol. 
+This will no longer work since each server must be able to find a valid license during the startup process.
+
```

Comment:
I don't have a strong opinion on the line spacing here but you might want to take a look at the preview at https://nomad-cij6qa2ft-hashicorp.vercel.app/docs/enterprise/license/faq#q-can-i-get-a-quick-summary-of-the-nomad-changes here because you need a double newline in order to break these into paragraphs.